src/DocRaptor.php

```diff
@@ -39,15 +40,21 @@ class ApiWrapper
     protected $baseurl;
 
 
-
     /**
      * @param string|null $api_key
+     * @param HttpTransferInterface $httpClient
      */
-    public function __construct($api_key = null)
+    public function __construct($api_key = null, HttpTransferInterface $httpClient = null)
```

Comment:
I think overall, it's neat now because the signature doesn't change, so if there are any old users, they can do the same as before (with the exception of composer autoloading and namespaces) but I can still skip the key and inject the client. (oops, I didn't tick the box that tests should be run before push, the tests are broken, need to adjust them) :-)